indra/newview/featuretable.txt

```diff
@@ -66,7 +66,8 @@ RenderDeferred				1	1
 RenderDeferredSSAO			1	1
 RenderShadowDetail			1	2
 RenderUseStreamVBO			1	1
-RenderFSAASamples			1	16
+RenderFSAAType			    1	2
+RenderFSAASamples			1	3
```

Comment:
Changing the meaning of a previous setting, so you should bump  the featuretable version number.  Otherwise, people who defaulted to low samples will suddenly default to high samples.